build/yamls/windows/base-containerd/conf/antrea-agent.conf

```diff
@@ -0,0 +1,134 @@
+# FeatureGates is a map of feature names to bools that enable or disable experimental features.
```

Comment:
This file is almost the same as the one in path "build/yamls/windows/base/conf/antrea-agent.conf", can we use a single file to maintain the general configurations?

Reply:
After discussion, Wenying and I agreed we can create a temporary folder for files we need during kustomize building, whether for containerd or docker windows yaml and we are going to introduce it in next PR. Do you have other suggestions? @tnqn

Reply:
Use sed to update kustomization.yml to avoid create a new folder.